cli/script/command-executor.ts

```diff
@@ -995,8 +995,9 @@ export var releaseCordova = (command: cli.IReleaseCordovaCommand): Promise<void>
         throw new Error("Platform must be either \"ios\" or \"android\".");
     }
 
+    log(chalk.cyan("Running \"cordova prepare\" command:\n"));
     try {
-        var prepareProcess: any = spawnSync("cordova", ["prepare", platform]);
+        var prepareProcess: any = spawnSync("cordova", ["prepare", platform, "--verbose"], { stdio: [0, 1, 2] });
```

Comment:
adding a comment will be helpful for stdio options.
process.stdin, process.stdout, process.stderr